lib/elixir/unicode/tokenizer.ex

```diff
@@ -76,9 +76,37 @@ defmodule String.Tokenizer do
       end
     end)
 
-  id_upper = letter_uptitlecase -- patterns
-  id_start = start -- patterns
-  id_continue = continue -- patterns
+  type_path = Path.join(__DIR__, "IdentifierType.txt")
+
+  restricted =
```

Comment:
37k items in here -- we could flip it around, keep the smaller list, and then subtract via

    (id_upper -- patterns) -- (id_upper -- allowed)

But it didn't feel like it made any difference ie when reloading in iex; obvs 37k item list in memory has some impact though depending on people w/use cases that involve 'compile elixir myself, on low-spec machine'.